internal/provider/client.go

```diff
@@ -165,8 +165,8 @@ func clientCreate(ctx context.Context, d *schema.ResourceData, meta interface{})
 		}
 
 		updateResourceFromClient(d, newOauthClient)
-		d.Set("client_identifier", newOauthClient.GetClientIdAlias())
-		d.SetId(newOauthClient.GetClientIdAlias())
+		d.Set("client_identifier", newOauthClient.GetClientId())
+		d.SetId(strconv.FormatInt(newOauthClient.GetClientId(), 10))
```

Comment:
I'm not entirely sure about the context, but didn't we previously deliberately change to using the client alias, because client ids will be different if a disaster-recovery type scenario ends up with a new server being built from scratch?

Either way it would probably be good if the commit message said why we were making this change.

Reply:
That is a good question. Let's discuss further details offline.

Reply:
I would mark this discussion as resolved

Reply:
> Does Authlete 3 honor the provided client id during creation api call?
Yes.
> If that is case, what is the usage of client_id_alias for Authlete 3? should we mark it as Computed and overload its value?
client_id_alias is used for OAuth, OpenID Connect ids. We shouldn't mark it as computed because we can set client_id_alias independently from client_id.